management/actions.go

```diff
@@ -200,6 +200,21 @@ type ActionExecution struct {
 	UpdatedAt *time.Time `json:"updated_at"`
 }
 
+// ActionLogSessionFilter defines a filter for the log session.
+type ActionLogSessionFilter struct {
+	Key string `json:"key"`
+	Val string `json:"val"`
+}
+
+// ActionLogSession contains a presigned URL that can be used for tailing realtime
+// logs from Actions.
+type ActionLogSession struct {
+	URL     *string    `json:"url,omitempty"`
+	Expires *time.Time `json:"expires,omitempty"`
+
+	Filters []ActionLogSessionFilter `json:"filters,omitempty"`
```

Comment:
I was able to find the json schema for this in api2 and filters has minItems: 1, so we can remove the omit empty here as well and from the other fields as well the pointers.

cc: @FadyMak

Reply:
@sergiught `filters` is not a required field so we need to have `omitempty` here otherwise it would result in a 400 if we pass in an empty `filters` array.

Reply:
Ok that makes sense, but what about URL and Expires? They seem to be required properties in the response so we can remove the pointers.